File: 2/2.js
function fiboEvenSum(n) {
  let a=1
  let b=2
  let ans=0
  while (a<=n){
    if (a%2==0)
    ans+=a
    let temp=a
    a=b
    b+=temp
  }
  console.log(ans)
 return ans
}
fiboEvenSum(10)

File: 2/2.py
import time

st = time.time()
lim = 4000000
a = 1
b = 2
ans = 0
while a <= lim:
    if a % 2 == 0:
        ans += a
    a, b = b, a + b
print('ans = ', ans)
print(time.time() - st, 's')
